Description: Some EPS files are rotated when I export the poster to a PDF file
Origin: upstream, r22053
Bug: https://bugs.scribus.net/view.php?id=14289
Bug-Debian: https://bugs.debian.org/837784

--- a/scribus/pdflib_core.cpp
+++ b/scribus/pdflib_core.cpp
@@ -7067,6 +7067,7 @@
 					opts.append( "-sProcessColorModel=/DeviceRGB" );
 				else
 					opts.append( "-sProcessColorModel=/DeviceCMYK" ); */
+				opts.append( "-dAutoRotatePages=/None" ); // #14289: otherwise EPS might come out rotated
 				if (convertPS2PDF(fn, tmpFile, opts) == 0)
 				{
 					imageLoaded = PDF_EmbeddedPDF(c, tmpFile, sx, sy, x, y, fromAN, Profil, Embedded, Intent, ImInfo, output);
